application/common/models/Email.php

```diff
@@ -98,18 +98,18 @@ public function send()
          */
         $log['status'] = 'sent';
         \Yii::info($log, 'application');
+        return 1;
     }
 
     /**
      * Attempt to send email and on failure update attempts count and save (queue it)
      * @throws \Exception
+     * @return int number of sent messages
      */
-    public function retry()
+    public function retry() : int
     {
         try {
-            $this->send();
-        } catch (EmailDelayedException $e) {
-            return 0;
+            return $this->send();
```

Comment:
Nice simplification 👍

Reply:
I think this was your suggestion a couple years ago. :)

Reply:
Whoops... didn't mean to pat myself on the back there.

Reply:
I think it's good that you're consistent!